### ISSUE TITLE
[bug] window.btc_providers shown as not installed

### ISSUE THREAD

**Issue by orenyomtov** [secretkeylabs/sats-connect#177]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### SATS-CONNECT Version

2.8.1

### Current Behavior

It shows providers in `window.btc_providers` as not installed

### Expected Behavior

Show installed providers as installed

### Anything else?

In
https://www.npmjs.com/package/@sats-connect/make-default-provider-config?activeTab=code
Change 
```
  config.options = config.options.concat(providers.filter((provider) => {
    return provider.id !== DefaultAdaptersInfo.xverse.id && provider.id !== "unisat";
  }).map((provider) => {
    return {
      ...provider,
      ...getInstallPrompt(provider)
    };
  }));
  return config;
  ```
  to
  ```
    config.options = config.options.concat(providers.filter((provider) => {
    return provider.id !== DefaultAdaptersInfo.xverse.id && provider.id !== "unisat";
  }).map((provider) => {
    return {
      ...provider,
      ...(provider.isInstalled ? {} : getInstallPrompt(provider))
    };
  }));
  return config;
  ```

**Comment by aryzing** [comment]
Thanks, PR up, should be fixed soon

**Comment by aryzing** [comment]
Should be fixed as of 2.8.2. Please let us know if you find any other issues :saluting_face:

**Comment by orenyomtov** [comment]
I pulled and rebuilt locally and it works!

FYI the Netlify deployments are failing to update https://sats-connect.netlify.app/
https://app.netlify.com/sites/sats-connect/deploys/66f402956c229b0008e0e9b0
